### PR TITLE
fix: send rewards to RewardEscrowV2

### PR DESCRIPTION
**Pull request by JChiaramonte7**

### CODE REVIEW

File: contracts/EscrowedMultipleMerkleDistributor.sol [hunk 1]
```diff
@@ -98,10 +98,11 @@ contract EscrowedMultipleMerkleDistributor is
         // mark it claimed and send the token to RewardEscrow
         _setClaimed(index, epoch);
         IERC20(token).approve(rewardEscrow, amount);
-        IRewardEscrow(rewardEscrow).createEscrowEntry(
+        IRewardEscrowV2(rewardEscrow).createEscrowEntry(
             account,
             amount,
-            52 weeks
+            52 weeks,
+            90
```
Comment (JaredBorders) [inline contracts/EscrowedMultipleMerkleDistributor.sol:105]: I know the fewer changes, the better, but there are a lot of raw numbers being used, like 52 weeks and 90, that would be better as constants.

**Review by JaredBorders — APPROVED** [review]
LGTM

Left a single comment about using constants. May be a good idea to at least comment where the `90` comes from (i.e. _earlyVestingFee is 90 because ...)

File: contracts/EscrowedMultipleMerkleDistributor.sol (hunk 1 above)
Comment (JChiaramonte7) [reply contracts/EscrowedMultipleMerkleDistributor.sol]: Did you one better, grabbed the constants directly from RewardEscrowV2

Comment (JaredBorders) [reply contracts/EscrowedMultipleMerkleDistributor.sol]: so clean ✨

**Pull request merged**